Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -29,6 +29,26 @@ def test_load_module_API(self):
         with self.assertRaises(ImportError):
             self.load_module('XXX')
 
+    def test_deprecation_warning_argument_name_and_fullname_use(self):
+        with self.assertWarns(DeprecationWarning) as cm:
+            self.machinery.ExtensionFileLoader(
+                    name=util.EXTENSIONS.name,
+                    path=util.EXTENSIONS.file_path)
+
+        self.assertIn("the 'name' parameter is deprecated; use 'fullname' "
+                      "instead", str(cm.warning))
```

Comment:
```suggestion
        self.assertIn("fullname", str(cm.warning))
```
See below for an explanation for this suggestion.